open_lab_notebook/Dez_update

```diff
@@ -0,0 +1,24 @@
+# Dezember
+
+## Flashback on last months
+- only got DICOM need to transform data into BIDS
+- manage to open Matlab files to get info about reccordings etc.
+
+## Week 1
+- wasn't available because of conference
+
+## Week 2
+- Corona hit me
+- searching for code to open Matlab-files-> used pymatreader -> did work, but no idea if that is what i needed
+
+## Week 3 
+- starting transformation DICOM into BIDS using [tutorial](https://reproducibility.stanford.edu/bids-tutorial-series-part-1a/)
```

Comment:
How do you think this is working? Does the tutorial provide enough information?

Reply:
as said in the meeting, it takes forever as it leads to a programm...so there has to be a smarter way.